hack/local-up-kubeedge.sh

```diff
@@ -167,12 +204,16 @@ kubectl create ns kubeedge
 create_device_crd
 create_objectsync_crd
 
+generate_streamserver_cert
+
 start_cloudcore
 
 sleep 2
 
 start_edgecore
 
+sudo iptables -t nat -A PREROUTING -p tcp --dport 10350 -j REDIRECT --to-port 10003
```

Comment:
Put this in function and check exit code.

Reply:
done